### DISCUSSION TITLE
How to uninstall?  (I need to see if Skia4Delphi is preventing a Delphi 11 november patch update)

### DISCUSSION THREAD

**Discussion by War3Evo** [skia4delphi/skia4delphi#175] [General]
Hi,

I used the Skia4Delphi_3.4.1_Setup.exe to install.

How do I uninstall?

**Comment by viniciusfbb** [comment]
Hi! You can remove it through [windows settings](https://support.microsoft.com/en-us/windows/uninstall-or-remove-apps-and-programs-in-windows-4b55f974-2cc6-2d2b-d092-5905080eaf98#ID0EBD=Windows_11) like any other app, or via GetIt (if you have installed using it).

But remember that if you install another version of Skia, the installer itself uninstalls the old versions. You can even install it always in the same folder.
